### PR TITLE
Add azure pipeline for windows nvda

### PR DESCRIPTION
**Pull request by outofambit**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208057616000774

### CODE REVIEW

File: .pipelines/windows-nvda.yml [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+      Write-Host "##vso[task.setvariable variable=NVDA_VERSION;]$env:p_nvda_version"
+
+- task: UseNode@1
+  displayName: Install Node
+  inputs:
+    version: '20.x'
+# - task: GoTool@0
+#   displayName: Install Go
+#   inputs:
+#     version: '1.10'
```
Comment (jugglinmike) [inline .pipelines/windows-nvda.yml:88]:
```suggestion
```

File: report-status-azure.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```
Comment (jugglinmike) [inline report-status-azure.sh:1]: It doesn't look like this script is used. Is that right?

**Review by jugglinmike — CHANGES_REQUESTED** [review]

File: .pipelines/windows-nvda.yml (hunk 1 above)
Comment (outofambit) [reply .pipelines/windows-nvda.yml]: good catch!

File: report-status-azure.sh (hunk 2 above)
Comment (outofambit) [reply report-status-azure.sh]: oh, yes, correct!

**Review by jugglinmike — APPROVED** [review]
Lookin good!

**Pull request merged**